src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -554,4 +554,15 @@ public function testArrayNotBeeingOverwritten()
         $this->assertSame('baz', $this->propertyAccessor->getValue($object, 'publicAccessor[value2]'));
         $this->assertSame(array('value1' => 'foo', 'value2' => 'baz'), $object->getPublicAccessor());
     }
+
+    /**
+     * @expectedException \Symfony\Component\PropertyAccess\Exception\InvalidArgumentException
+     * @expectedExceptionMessage Expected argument of type "Countable", "string" given
```

Comment:
I would have expected a test checking for the new "must implement interface" string in the exception message.